File: e-commerce/src/main/java/com/example/e_commerce/category/entity/Category.java
package com.example.e_commerce.category.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Category {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long categoryId;
    private String categoryName;
    private String imageName;
    private String imageType;
    @Lob
    private byte[] imageData;
}

File: app/src/main/java/com/example/bytebasket/category/Category.kt
package com.example.bytebasket.category

import android.os.Parcelable
import kotlinx.parcelize.Parcelize


@Parcelize
data class Category(
    val categoryId: Long?,
    val categoryName: String,
    var imageName: String?,
    val imageType: String?,
    val imageData: String?
):Parcelable
